packages/core/src/components/cv-data-table/cv-data-table.vue

```diff
@@ -324,10 +324,10 @@ export default {
   methods: {
     checkSlots() {
       // NOTE: this.$slots is not reactive so needs to be managed on beforeUpdate
-      this.hasBatchActions = this.$slots['batch-actions'];
-      this.hasActions = this.$slots.actions;
-      this.hasToolbar = this.$slots.actions || this.$listeners.search || this.$slots['batch-actions'];
-      this.isHelper = this.$slots['helper-text'] !== undefined || (this.helperText && this.helperText.length);
+      this.hasBatchActions = !!this.$slots['batch-actions'];
```

Comment:
In other places, e.g. `cv-side-nav-menu`, you use comparison with undefined `this.$slots['nav-icon'] !== undefined`. I think it makes sense to have it consistent.